doc/languages-frameworks/python.section.md

```diff
@@ -1632,3 +1632,25 @@ would be:
 ```ShellSession
 $ maintainers/scripts/update-python-libraries --target minor --commit --use-pkgs-prefix pkgs/development/python-modules/**/default.nix
 ```
+
+## CPython Update Schedule
+
+With [PEP 602](https://www.python.org/dev/peps/pep-0602/), CPython now
+follows a yearly release cadence. In nixpkgs, all supported interpreters
+are made availaable, but only the most recent two
+interpreters package sets are built; this is a compromise between being
+the latest interpreter, and what the majority of the python packages support.
+
+New CPython interpreters are released in October. Generally, it takes some
+time for the majority of active python projects to support the latest stable
+interpreter. To help ease the migration for Nixpkgs users
+between python interpreters the schedule below will be used:
+
+| When | Event |
+| --- | --- |
+| After YY.11 Release | Bump CPython package set window. The latest and previous latest stable should now be built. |
```

Comment:
So with `.11` the latest Python is not the `python3` attribute but in `05` it is? Seems a good idea to me.

Reply:
Yep, that's the idea. The `after` wording is to allow for some additional time if there's some fundamental issues which exist in the community.